File: 07_Tues/Cardio.java
import java.util.Scanner;

public class Cardio {
    public static void main(String[] args){
        Integer[] ints = {1, 0, 0, 0};
        solution2(ints);
    }

    public static void solution1(Integer[] ints){
        int ones = 0;
        int zeros = 0;
        for (int i : ints){
            if (i == 1){
                ones++;
            }else if (i == 0){
                zeros++;
            }
        }

        System.out.println(isMoreOnes(ones, zeros));
    }

    public static void solution2(Integer[] ints){
        int ones = 0;
        int zeros = 0;
        int i = 0;
        while (i < ints.length){
            if (i == 1){
                ones++;
            }else if (i == 0){
                zeros++;
            }
            i++;
        }

        System.out.println(isMoreOnes(ones, zeros));
    }


    public static boolean isMoreOnes( int ones, int zeros){
        if (ones > zeros){
            return true;
        }
        return false;
    }
}

File: 07_Tues/cardio.py
def main():
    ints = [1, 1, 0]
    solution_1(ints)

def solution_1(ints):
    ones = 0
    zeros = 0

    for i in ints:
        if i == 1:
            ones += 1
        elif i ==0:
            zeros += 1
    print(ones > zeros)

def solution_2(ints):
    ones = 0
    zeros = 0
    i=0

    while i < len(ints):
        if i == 1:
            ones += 1
        elif i ==0:
            zeros += 1
        i += 1
    print(ones > zeros)
    

def return_largest(largest, current_int):
    if largest < current_int:
        return current_int    
    return largest

def return_smallest(smallest, current_int):
    if smallest > current_int:
        return current_int    
    return smallest

if __name__ == "__main__":
    main()
